README.md

```diff
@@ -21,6 +25,40 @@ To test the complete result from a nix-build:
     $ nix-build
     $ nix-shell --run 'cd result/ && python -m http.server'
 
+### Docker Desktop for Mac
+
+Unfortunately it will not build on macOS natively. 
+But it can be build in a [Docker container on macOS](https://www.docker.com/products/docker-desktop). 
+(You may need to increase the maximum "Disk image size" in Docker's Resources Preferences.)
+
+In your favourite macOS terminal:
+
+    $ docker run -p 8000:8000 -it --name nixos-homepage nixos/nix
+
+Since git, fd and entr are not installed in the container:
+
+    # nix-shell -p git fd entr
+
+Following with the same steps as on Linux/NixOS:
+
+    [nix-shell]$ git clone https://github.com/NixOS/nixos-homepage.git
+    [nix-shell]$ cd nixos-homepage
+    [nix-shell]$ nix-shell
+    [nix-shell]$ make
+
+Add the `--bind 0.0.0.0` argument to bind the web server with the container's external interface too:
+
+    [nix-shell]$ python -m http.server --bind 0.0.0.0
+
+Then open http://127.0.0.1:8000/index.html in your browser on macOS.
+
+A single terminal screen is inconvenient for serious development, so you may want to set up openssh on the container to access the sources with the IDE on your Mac.
```

Comment:
I would skip this section and try to provide instructions to only run one instance of container with running `nix-shell -p` outside the container.